feat(changelog): containers-changed-serverless-containers-default-sandbo 2024-09-19 (#3727)
scaleway/docs-content: changelog/september2024/2024-09-19-containers-changed-serverless-containers-default-sandbo.mdx

---

> title: Serverless Containers default sandbox set to V2

Reviewer: ```suggestion
title: Serverless Containers default sandbox set to v2
```

status: changed
author:
    fullname: 'Join the #serverless-containers channel on Slack.'
    url: 'https://slack.scaleway.com'
date: 2024-09-19
category: serverless
product: containers
---

> To reduce cold-starts we recently added a new sandbox v2.

Reviewer: ```suggestion
To reduce cold-starts, we recently added a new version of our sandbox: v2.
```

> Now on the console it's the default value for new Containers, we encorage to use it for better performances.

Reviewer: ```suggestion
This is now the default sandbox version for new Functions on the console, as it offers enhanced performance.
```
